packages/dmn-editor/tests/e2e/__fixtures__/propertiesPanel/decisionProperties.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { PropertiesPanelBase } from "./propertiesPanelBase";
+
+export class DecisionProperties extends PropertiesPanelBase {
+  public async changeNodeQuestion(args: { nodeName: string; newQuestion: string }) {
```

Comment:
As the panel is from the Node, we don't need to add it to the name. And as we're using the `get`, we could use `set`. This way we would have `setQuestion`. The same applies to the other methods.

Reply:
Incorporated as part of https://github.com/ljmotta/kie-tools/pull/27